frontend/interfaces/host.ts

```diff
@@ -202,7 +203,8 @@ export interface IHost {
   users: IHostUser[];
   device_users?: IDeviceUser[];
   munki?: IMunkiData;
-  mdm?: IHostMdmData;
+  mdm_enrollment_status: MdmStatus;
+  mdm_server_url: string;
```

Comment:
Just a heads up, in the next sprint, we will be moving these properties to their own object:

```json
"mdm": {
  "enrollment_status": "Off",
  "server_url": "https://www.example.com/1"
}
```